### PR TITLE
@bhoggard: updates to latest gris and other gems

### PR DESCRIPTION
**Pull request by dylanfareed**
This PR updates to the latest versions of the Gris gem (and really all gems that are available for update).

Since we initially set this app up we've moved token auth and pagination out into their own helper gems. So this also adds `gris_paginator` and `gris-toko_ohno` to the mix.

### CODE REVIEW

File: app/api/v1/presenters/root_presenter.rb [hunk 1]
```diff
@@ -3,76 +3,16 @@ module Presenters
...
-      private
-
-      def base_url(opts)
-        request = Grape::Request.new(opts[:env])
-        request.base_url
-      end
+      resource_links :account
+      resource_links :feature, [:account_id]
+      resource_links :message, [:account_id]
+      resource_links :route, [:account_id]
```
Comment (dylanfareed) [inline app/api/v1/presenters/root_presenter.rb]: Newer & simpler syntax for building basic links in presenters.

**Comment by dylanfareed** [comment]
Gonna re-assign this to @bhoggard as he's also working in Echo at the moment.

File: app/api/v1/presenters/root_presenter.rb (hunk 1 above)
Comment (orta) [inline app/api/v1/presenters/root_presenter.rb]: it sure is, lots of deletions here :+1:

File: config/database.yml [hunk 2]
```diff
@@ -2,8 +2,8 @@ default: &default
   adapter: postgresql
   encoding: unicode
   host: localhost
-  username: <%= ENV['DB_USER'] %>
-  password: <%= ENV['DB_PASS'] %>
+  username: <%= ENV['DATABASE_USER'] %>
+  password: <%= ENV['DATABASE_PASS'] %>
```
Comment (orta) [inline config/database.yml]: did you remember to update the heroku ENVs

Comment (dylanfareed) [inline config/database.yml]: Heroku ignores this altogether. So we are good there.

File: app/api/v1/presenters/root_presenter.rb [hunk 3]
```diff
@@ -3,76 +3,16 @@ module Presenters
...
-
-      private
-
-      def base_url(opts)
-        request = Grape::Request.new(opts[:env])
-        request.base_url
-      end
+      resource_links :account
+      resource_links :feature, [:account_id]
+      resource_links :message, [:account_id]
```
Comment (bhoggard) [inline app/api/v1/presenters/root_presenter.rb]: Should this be `:account_id` or `:id`?

Comment (dylanfareed) [inline app/api/v1/presenters/root_presenter.rb]: `account_id`. Its a replacement for the uri template options previously listed here (page and size come by default) - https://github.com/artsy/echo/pull/19/files#diff-d2263ae86a9a4d5fe8206564f28b5363L30

**Pull request merged**